tests/src/test_generate_mapping.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import config
+from google.cloud import bigquery
+import generate_td_to_bq_mapping as gm
+import os
+from  pathlib import Path
+import utils
+from google.api_core.exceptions import NotFound, Forbidden, BadRequest, ServiceUnavailable, Conflict, TooManyRequests
+
+class TestObjectMapping:
+
+    def test_get_created_tables_and_views_success(self,
+                                                  create_sql_file):
+        file_path = "test_file.sql"
+        sql_path = Path(file_path)
+        get_tables_and_views = gm.get_created_tables_and_views(sql_path)
+        assert type(get_tables_and_views) == type(list())
```

Comment:
This should also evaluate the tables/views that get returned to make sure they are the ones that we expect.